app/controllers/granules_controller.rb

```diff
@@ -42,7 +42,10 @@ def timeline
 
   class GranuleUrlStreamer
```

Comment:
I'd prefer to see two completely separate URL streamers, one for CMR and one for CWIC (`CmrGranuleUrlStreamer`, `CwicGranuleUrlStreamer`). They probably deserve their own files. In general, we should limit the amount that we have to use if-logic to distinguish between CWIC and CMR and use methods or classes instead.